src/components/Player.vue

```diff
@@ -640,14 +664,50 @@ li.move:not(.from) .player .overlay svg.move {
   border-radius: 10px;
   top: 5px;
   box-shadow: 0 0 5px black;
-  text-overflow: ellipsis;
-  overflow: hidden;
   padding: 0 4px;
 
+  span {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
   #townsquare:not(.spectator) &:hover,
   &.active {
     color: red;
   }
+
+  &:hover .pronouns {
+    opacity: 1;
+    color: white;
+  }
+
+  .pronouns {
+    display: inline-flex;
+    position: absolute;
+    right: 120%;
```

Comment:
```suggestion
    right: 110%;
```
Distance is a tad too big, I'd say.